tests/php/phpstan/bootstrap.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
```

Comment:
This file will be shipped in the final module zip. Can we protect it somehow ? Like the `app_dev.php` file of Symfony cant be used on prod env ?

Reply:
@matks How is generated the ZIP ? Can't exclude some directories (like .gitignore) ?

Reply:
The zip generation is handled by the Platforms team who built it then make it available through Addons API. I dont think we can modify it in the close future 😐

Reply:
I tried in detecting if users are in CLI mode like Symfony.

Reply:
@matks Fixed in last commits